docs/Displaying a User Interface/Alerts/index.md

```diff
@@ -379,3 +468,15 @@ cancelInteraction.setOnRPCResponseListener(new OnRPCResponseListener() {
 sdlManager.sendRPC(cancelInteraction);
 ```
 !@  
+
+@![javascript]
+```js
+const cancelInteraction = new SDL.rpc.messages.CancelInteraction().setFunctionIDParam(SDL.rpc.enums.FunctionID.Alert);
+sdlManager.addRpcListener(SDL.rpc.enums.FunctionID.CancelInteraction, function (response) {
+    if (response instanceof SDL.rpc.RpcResponse && response.getSuccess()) {
+        console.log('Alert was dismissed successfully');
+    }
+});
```

Comment:
To convert to async/await style of getting responses from RPC request. Check other instances in this PR